src/modules/auth/login.controller.ts

```diff
@@ -105,7 +106,7 @@ export class LoginController {
     );
 
     const [token, refreshToken] = await Promise.all([
-      this.tokenService.generateAccessToken({ did: userDid, verifiedRoles }),
+      this.tokenService.generateAccessToken({ did: userDid, verifiedRoles, supportedOrigins, supportedHosts }),
```

Comment:
We don't want to restrict clients to specific "supportedOrigins". I think instead the approach that we should take is that we store the origin that the client used during login and check if that is the origin of subsequent requests.
I think this is roughly what we need:
```suggestion
      const origin = req.headers['origin'];
      this.tokenService.generateAccessToken({ did: userDid, verifiedRoles, origin }),
```